source/features/follow-file-renames.tsx

```diff
@@ -27,7 +27,7 @@ function init(): false | void {
 		return false;
 	}
 
-	disabledPagination.forEach(async button => {
+	for (const button of disabledPagination) {
```

Comment:
Am I correct the using `forEach` takes quicker here since it does not respect the await?

Reply:
```suggestion
	for (const button of disabledPagination) {
	    void linkify(button, url);
	}
```

As I had suggested

Reply:
Ahhh missed that.